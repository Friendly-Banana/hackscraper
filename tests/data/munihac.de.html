<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
    <title>MuniHac 2024</title>
    <link href="./assets/styles.css" rel="stylesheet" />
    <link href="./assets/custom.css" rel="stylesheet" />
    <link href="./assets/bootstrap-icons.css" rel="stylesheet" />
    <link rel="apple-touch-icon" sizes="180x180" href="./favicon/apple-touch-icon-180x180.png" />
    <link rel="shortcut icon" type="image/x-icon" href="./favicon/favicon-32x32.ico" />
    <link rel="icon" type="image/png" sizes="96x96" href="./favicon/favicon-96x96.png" />
    <meta name="msapplication-square310x310logo" content="/favicon/mstile-310x310.png" />
    <meta name="msapplication-TileColor" content="#ffffff" />
  </head>
  <body id="page-top">
     <nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top" id="mainNav">
  <div class="container px-4">
    <a class="navbar-brand" href="./2024.html#page-top">
      <img alt="MuniHac 2024" src="./img/2024/munihac-2024-logo-oneline.svg" style="height: 1em; vertical-align: bottom" />
    </a>
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>
    <div class="collapse navbar-collapse" id="navbarResponsive">
      <ul class="navbar-nav ms-auto">
        <li class="nav-item"><a class="nav-link" href="./2024.html#about">About</a></li>
        <li class="nav-item"><a class="nav-link" href="./2024.html#schedule">Schedule</a></li>
        <li class="nav-item"><a class="nav-link" href="./2024.html#programme">Programme</a></li>
        <li class="nav-item"><a class="nav-link" href="./2024.html#projects">Projects</a></li>
        <li class="nav-item"><a class="nav-link" href="./2024.html#contact">Contact</a></li>
        <li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="dropdown-archive" data-bs-toggle="dropdown" aria-expanded="false">Archive</a>
  <ul class="dropdown-menu" aria-labelledby="dropdown-archive">
    <li><a class="dropdown-item" href="2016.html">MuniHac 2016</a></li>
    <li><a class="dropdown-item" href="2018.html">MuniHac 2018</a></li>
    <li><a class="dropdown-item" href="2019.html">MuniHac 2019</a></li>
    <li><a class="dropdown-item" href="2020.html">MuniHac 2020</a></li>
    <li><a class="dropdown-item" href="2022.html">MuniHac 2022</a></li>
    <li><a class="dropdown-item" href="2023.html">MuniHac 2023</a></li>
    <li><a class="dropdown-item" href="2024.html">MuniHac 2024</a></li>
  </ul>
</li>

        <li class="nav-item">
          <a class="nav-link" href="https://twitter.com/MuniHac" target="_tab"><i class="bi bi-twitter-x"></i></a>
        </li>
      </ul>
    </div>
  </div>
</nav>
  <header>
  <div class="container px-4 pb-4">
    <div class="row">
      <div class="col-md-2"></div>
      <div class="col-md-8">
        <!-- TODO: Use new logo provided by Franz Thoma -->
        <img alt="MuniHac 2024" src="./img/2024/munihac-2024-logo.svg" style="width: 100%" />
      </div>
      <div class="col-md-2"></div>
    </div>
  </div>
  <div class="text-center text-light bg-dark">
    <div class="container px-4 py-5">
      <h1 class="display-1">MuniHac 2024</h1>
      <p class="lead">October 11â13, 2024 â¢ on-site in Munich</p>
      <div class="d-flex flex-row flex-wrap justify-content-center mb-2">
        <a class="rounded-pill px-3 mx-2 mb-2 btn btn-outline-light" href="./2024.html#about">Learn more</a>
      </div>
    </div>
  </div>
</header>


<a name="about"></a>
<div class="container px-4 py-4">
  <div class="row">
    <div class="col-md-8">
      <h1>About MuniHac</h1>
      <p>
        MuniHac is an annual three-day hackathon and conference in Munich that brings together Haskell developers and
        enthusiasts from across the globe. This event offers a unique opportunity to improve your Haskell skills,
        network with experts in the community, and participate in a range of workshops, talks, and projects for both
        beginners and experienced developers.
      </p>

      <p>
        The MuniHac is organized by
        <a href="https://www.tngtech.com/en/index.html">TNG Technology Consulting GmbH</a> and will be hosted at the TNG
        offices in Munich (Beta-StraÃe 13, 85774 UnterfÃ¶hring).
      </p>
    </div>

    <div class="col-md-4">
      <iframe width="100%" height="254px" frameborder="0" style="border:0" src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2660.059811205255!2d11.652588151972282!3d48.18619907912526!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x479e74c74cfca785%3A0xb774bd7390f72580!2sTNG+Technology+Consulting+GmbH!5e0!3m2!1sde!2sde!4v1566341898872!5m2!1sde!2sde" allowfullscreen></iframe>
      <address>
      TNG Technology Consulting GmbH<br />
      Beta-StraÃe 13<br />
      85774 UnterfÃ¶hring
      </address>
      </div>
    </div>
  </div>
</div>


<a name="schedule"></a>
<div class="bg-gray-light"><div class="container px-4 py-5">
  <h2>Schedule</h2>

  <p>Please note that all times are CEST.</p>

  <!--
  Here we are abusing and mis-using the existing CSS class:
    - hacking: used for "hacking" and "food" events
    - talk: used for talks (except keynotes) and workshops
    - workshop: used for keynotes
  -->

  <table class="table table-bordered schedule">
    <tr class="bg-primary">
      <th colspan="3">Friday, Oct. 11th 2024</th>
    </tr>
    <tr>
      <th scope="col" class="col-md-1">Time</th>
      <th scope="col" class="col-md-3 hacking">Hackathon</th>
      <th scope="col" class="col-md-4 workshop">Talks & Workshops</th>
    </tr>
    <tr>
      <th scope="row">8:30</th>
      <td colspan="2">Doors open &amp; Registration</td>
    </tr>
    <tr>
      <th scope="row">9:00</th>
      <td colspan="2" class="hacking">Opening</td>
    </tr>
    <tr>
      <th scope="row">9:15</th>
      <td colspan="2" class="hacking">Project Marketplace</td>
    </tr>
    <tr>
      <th scope="row">09:30</th>
      <td class="hacking">Hack on</td>
      <td class="workshop">Beginners Workshop</td>
    </tr>
    <tr>
      <th scope="row">11:30</th>
      <td colspan="2" class="text-center talk">
        Keynote:
        <a href="2024.html#HannesSiebenhandl">Hannes Siebenhandl: <i>Exploring Haskell Language Server via the Cabal Plugin</i></a>
      </td>
    </tr>
    <tr>
      <th scope="row">12:30</th>
      <td colspan="2" class="hacking">Lunch</td>
    </tr>
    <tr>
      <th scope="row">13:30</th>
      <td class="hacking">Hack on</td>
      <td class="workshop">Beginners Workshop (continued)</td>
    </tr>
    <tr>
      <th scope="row">17:30</th>
      <td colspan="2">Official end</td>
    </tr>
  </table>

  <br />

  <table class="table table-bordered schedule">
    <tr class="bg-primary">
      <th colspan="3">Saturday, Oct. 12th 2024</th>
    </tr>
    <tr>
      <th scope="col" class="col-md-1">Time</th>
      <th scope="col" class="col-md-3 hacking">Hackathon</th>
      <th scope="col" class="col-md-4 workshop">Talks & Workshops</th>
    </tr>
    <tr>
      <th scope="row">9:00</th>
      <td colspan="2">Doors open</td>
    </tr>
    <tr>
      <th scope="row">9:30</th>
      <td class="hacking">Hack on</td>
      <td class="workshop">
        (Talk)
        <a href="2024.html#VanessaMcHale">Vanessa McHale: <i>Jacinda: Compiling a Functional AWK Replacement Efficiently</i></a>
      </td>
    </tr>
    <tr>
      <th scope="row">10:30</th>
      <td class="hacking">Hack on</td>
      <td class="workshop">
        (Talk)
        <a href="2024.html#AndrewLelechenko">Andrew Lelechenko: <i>CLC activities report</i></a>
      </td>
    </tr>
    <tr>
      <th scope="row">11:30</th>
      <td colspan="2" class="text-center talk">
        Keynote: <a href="2024.html#Hecate">HÃ©cate: <i>Effect Systems in Practice</i></a>
      </td>
    </tr>
    <tr>
      <th scope="row">12:30</th>
      <td colspan="2" class="hacking">Lunch</td>
    </tr>
    <tr>
      <th scope="row">13:30</th>
      <td class="hacking">Hack on</td>
      <td class="workshop">
        (Workshop)
        <a href="2024.html#DominikSchrempf">Dominik Schrempf: <i>Statistical inference with Markov chain Monte Carlo algorithms</i></a>
      </td>
    </tr>
    <tr>
      <th scope="row">15:30</th>
      <td class="hacking">Hack on</td>
      <td class="workshop">
        (Workshop)
        <a href="2024.html#MartinValdesDeLeon">Martin Valdez de Leon: <i>Generative Music with Haskell</i></a>
      </td>
    </tr>
    <tr>
      <th scope="row">17:30</th>
      <td colspan="2">Official end</td>
    </tr>
  </table>

  <br />

  <table class="table table-bordered schedule">
    <tr class="bg-primary">
      <th colspan="3">Sunday, Oct. 13th 2024</th>
    </tr>
    <tr>
      <th scope="col" class="col-md-1">Time</th>
      <th scope="col" class="col-md-3 hacking">Hackathon</th>
      <th scope="col" class="col-md-4 workshop">Talks & Workshops</th>
    </tr>
    <tr>
      <th scope="row">9:00</th>
      <td colspan="2">Doors open</td>
    </tr>
    <tr>
      <th scope="row">09:30</th>
      <td class="hacking">Hack on</td>
      <td class="workshop" rowspan="2">
        (Workshop)
        <a href="2024.html#IoannisKatsaros">Ioannis Katsaros: <i>You don't need to be smart to use Haskell</i></a>
      </td>
    </tr>
    <tr>
      <th scope="row">10:00</th>
      <td class="hacking">Bavarian Breakfast</td>
    </tr>
    <tr>
      <th scope="row">11:30</th>
      <td class="hacking" colspan="2">Bavarian Breakfast</td>
    </tr>
    <tr>
      <th scope="row">12:00</th>
      <td colspan="2" class="text-center talk">
        Keynote: <a href="2024.html#NikiVazou">Niki Vazou: <i>Verification with Refinement Types</i></a>
      </td>
    </tr>
    <tr>
      <th scope="row">13:00</th>
      <td colspan="2" class="hacking">Hack on</td>
    </tr>
    <tr>
      <th scope="row">14:30</th>
      <td class="hacking" colspan="2">Project presentations</td>
    </tr>
    <tr>
      <th scope="row">17:00</th>
      <td colspan="2">Doors close</td>
    </tr>
  </table>
</div>
</div>

<a name="programme"></a>
<div class="container px-4 py-5">
  <h2>Programme</h2>

  <div class="row py-3">
    <div class="col-md-12">
      <a name="NikiVazou"></a>
      <div class="d-flex flex-row flex-wrap">
        <img src="img/2024/Niki_Vazou.jpg" class="rounded-circle me-4 mb-2" style="height: 96px; width: 96px; object-fit: cover" />
        <div>
          <h5>Keynote</h5>
          <h3>Liquid Haskell: Verification with Refinement Types</h3>
          <h4>Niki Vazou</h4>
        </div>
      </div>
    </div>
    <div class="col-md-12">
      <div class="youtube-button-container youtube-2024-niki float-end" style="margin-left: 30px; margin-bottom: 10px">
        <button onclick="loadYoutubeIframe('7m7T3hCq5ZY', this)" class="youtube-button">Load video</button>
      </div>
      <p>
        Refinement types decorate the types of a programming language with logical predicates to allow more expressive
        type specifications. Originally, refinement type based specifications were restricted to SMT decidable theories
        and allowed automatic âlightâ verification, for example properties like non-division by zero or in-bound
        indexing. Verification of such light properties though requires âdeeperâ specifications, for example âis append
        associative?â In this talk, we will interactively see how to use the refinement type checker of Liquid Haskell
        to verify Haskell programs.
      </p>
      <p><em>This work is funded by the ERC project CRETE.</em></p>
    </div>
  </div>

  <div class="row py-3">
    <div class="col-md-12">
      <a name="HannesSiebenhandl"></a>
      <div class="d-flex flex-row flex-wrap">
        <img src="img/2024/Hannes_Siebenhandl.jpg" class="rounded-circle me-4 mb-2" style="height: 96px; width: 96px; object-fit: cover" />
        <div>
          <h5>Keynote</h5>
          <h3>Exploring Haskell Language Server via the Cabal Plugin</h3>
          <h4>Hannes Siebenhandl</h4>
        </div>
      </div>
    </div>
    <div class="col-md-12">
      <div class="youtube-button-container youtube-2024-siebenhandl float-end" style="margin-left: 30px; margin-bottom: 10px">
        <button onclick="loadYoutubeIframe('5mp_Yi6sGwI', this)" class="youtube-button">Load video</button>
      </div>
      <p>
        The <code>.cabal</code> file defines what encompasses a package in the Haskell ecosystem. It is strictly
        required if you want to upload your package to Hackage. As such, almost anyone has to write or modify
        <code>.cabal</code>
        files at some point in their Haskell career, but due to its unique syntax, this often proves harder than
        necessary. Recently, Haskell Language Server started to close the gap between support for Haskell and cabal
        source files, by introducing the cabal-plugin.
      </p>
      <p>
        In this talk, we are going to take a closer look at the integration of the cabal-plugin into the architecture of
        the Haskell Language Server and explore its key concepts for providing a snappy, interactive developer
        experience. We are going to discuss the Language Server Protocol, and the plugin architecture, which allows us
        to extend HLS with a minimal amount of effort. These features will be explained by doing a deep-dive into the
        "cabal-plugin", which allows HLS to check <code>.cabal</code> files for syntax errors, provide completions, and
        even more features that help to read, write and modify <code>.cabal</code> files
      </p>
    </div>
  </div>

  <div class="row py-3">
    <div class="col-md-12">
      <a name="Hecate"></a>
      <div class="d-flex flex-row flex-wrap">
        <img src="img/2024/Hecate.png" class="rounded-circle me-4 mb-2" style="height: 96px; width: 96px; object-fit: cover" />
        <div>
          <h5>Keynote</h5>
          <h3>Effect Systems in Practice</h3>
          <h4>HÃ©cate</h4>
        </div>
      </div>
    </div>
    <div class="col-md-12">
      <div class="youtube-button-container youtube-2024-hecate float-end" style="margin-left: 30px; margin-bottom: 10px">
        <button onclick="loadYoutubeIframe('lRU6TDgadqw', this)" class="youtube-button">Load video</button>
      </div>
      <p>
        With HÃ©cate, we will approach the topic of Effect Systems to write Haskell applications, understand why it is
        important do have semantic effect tracking in your software, and see how it all fits into the design and
        maintenance of software systems. We will use Effectful, developed for industrial needs of performance and
        ease-of-use.
      </p>
    </div>
  </div>

  <div class="row py-3">
    <div class="col-md-12">
      <a name="AndrewLelechenko"></a>
      <div class="d-flex flex-row flex-wrap">
        <img src="img/2024/andrew_lelechenko.jpeg" class="rounded-circle me-4 mb-2" style="height: 96px; width: 96px; object-fit: cover" />
        <div>
          <h5>Talk</h5>
          <h3>CLC activities report</h3>
          <h4>Andrew Lelechenko</h4>
        </div>
      </div>
    </div>
    <div class="col-md-12">
      <p>
        In December I'll be wrapping up my 3-year tenure of the CLC chair and I'd like to present a CLC activities
        report for this period. Brief introduction to CLC structure, roles and responsibilities; case studies for past
        CLC proposals; crash course "how to get your proposal approved"; preparations for PVP 1.1; review of core
        libraries. I hope to demistify the work of the committee and inspire people to apply for vacant positions early
        next year.
      </p>
    </div>
  </div>

  <div class="row py-3">
    <div class="col-md-12">
      <a name="VanessaMcHale"></a>
      <div class="d-flex flex-row flex-wrap">
        <img src="img/2024/vanessa-mchale.png" class="rounded-circle me-4 mb-2" style="height: 96px; width: 96px; object-fit: cover" />
        <div>
          <h5>Talk</h5>
          <h3>Jacinda: Compiling a Functional AWK Replacement Efficiently</h3>
          <h4>Vanessa McHale</h4>
        </div>
      </div>
    </div>
    <div class="col-md-12">
      <p>
        Jacinda is a laconic functional language for stream processing on the command-line. The functional style for
        stream processing is appealing â maps and folds express stream processing more directly than variable updates â
        but it requires some cleverness on the backend to make this efficient. I'll show off the Jacinda language, and
        how to translates this into something for efficient evaluation. Along the way, I'll show off Haskell features
        for implementing languages that make, and some of the tricks for writing a modern language that didn't make it
        into the textbook.
      </p>
    </div>
  </div>

  <div class="row py-3">
    <div class="col-md-12">
      <a name="MartinValdesDeLeon"></a>
      <div class="d-flex flex-row flex-wrap">
        <img src="img/2024/martin-valdes-de-leon.jpg" class="rounded-circle me-4 mb-2" style="height: 96px; width: 96px; object-fit: cover" />
        <div>
          <h5>Workshop</h5>
          <h3>Generative Music with Haskell</h3>
          <h4>Martin Valdes de Leon</h4>
        </div>
      </div>
    </div>
    <div class="col-md-12">
      <p>Have you ever wondered how synthesizers work? How about building your own, using Haskell?</p>
      <p>
        In this workshop we'll introduce some basic notions of sound synthesis, and will take a look at how to apply
        these concepts to build different kinds of simple musical instruments, and ultimately make some functional
        music.
      </p>
      <p>
        This workshop is aimed at Haskell beginners, and requires no prior musical knowledge. Participants are expected
        to have their own development environments already configured.
      </p>
    </div>
  </div>

  <div class="row py-3">
    <div class="col-md-12">
      <a name="IoannisKatsaros"></a>
      <div class="d-flex flex-row flex-wrap">
        <img src="img/2024/Ioannis_Katsaros.jpg" class="rounded-circle me-4 mb-2" style="height: 96px; width: 96px; object-fit: cover" />
        <div>
          <h5>Workshop</h5>
          <h3>You don't need to be smart to use Haskell</h3>
          <h4>Ioannis Katsaros</h4>
        </div>
      </div>
    </div>
    <div class="col-md-12">
      <p>
        â
        <i>"Sure, Haskell is powerful and safe, but there's no way an idiot like me can build anything useful with it,
          right?"</i>
      </p>
      <p>That was me three years ago and maybe you're in the same place now; but this is an illusion.</p>
      <p>
        To deliver us from this kind of negative self-talk, we'll build a small http server using the servant API and
        through example, realize there's no need to be a wizard to write useful Haskell applications.
      </p>
      <p>This workshop is aimed at Haskell beginners, basic knowledge of HTTP APIs is welcome but not necessary.</p>
      <p>
        Participants are expected to have their own development environments configured, but a gitspace is being worked
        on for the project and may be available for those who don't have either the knowledge or the time to set up
        their environment.
      </p>
    </div>
  </div>

  <div class="row py-3">
    <div class="col-md-12">
      <a name="DominikSchrempf"></a>
      <div class="d-flex flex-row flex-wrap">
        <img src="img/2024/Dominik_Schrempf.jpg" class="rounded-circle me-4 mb-2" style="height: 96px; width: 96px; object-fit: cover" />
        <div>
          <h5>Workshop</h5>
          <h3>Statistical inference with Markov chain Monte Carlo algorithms</h3>
          <h4>Dominik Schrempf</h4>
        </div>
      </div>
    </div>
    <div class="col-md-12">
      <p>
        Haskell excels in finding programming errors during compile time. We also know Haskell for being lazy and for
        treating values as immutable. Lazyness and immutability are likely reasons why numerical and statistical
        analysis of data have never been a particular strength of Haskell.
      </p>
      <p>
        I have used Haskell extensively during my career as a researcher and have tried to combine the benefits of the
        strong type system of Haskell with blazing-fast statistical estimators.
      </p>
      <p>
        In this workshop, we will have a look at a specific class of algorithms used for statistical inference: Markov
        chain Monte Carlo algorithms. We will get a basic understanding of such algorithms, and, discuss how Haskell can
        aid us in avoiding common user errors. In particular, we will use the Haskell library <code>mcmc</code> to
        perform some exemplary analyses
      </p>
    </div>
  </div>
</div>


<a name="projects"></a>
<div class="bg-gray-light"><div class="container px-4 py-5">
  <h3>Projects</h3>

  <p>If you want to host another project, you'll have the opportunity to pitch it in the Project Marketplace on Friday morning, and we'll add it to the list!</p>

  <ul>
    <li><b><a href="https://github.com/kuribas/hasqlator-mysql">hasqlator</a>:</b> Porting a mid-level SQL generation library to Postgres</li>
    <li><b><a href="https://flora.pm">flora.pm</a>:</b> The meta-index for Haskell packages <span class="badge bg-primary">beginner-friendly</span></li>
    <li><b>Stackage:</b> Dependency bounds automation</li>
    <li><b><a href="https://github.com/haskell/haskell-language-server/">Haskell Language Server <span class="badge bg-primary">beginner-friendly</span></a></b></li>
    <li><b><a href="https://github.com/MangoIV/cabal-audit">cabal-audit</a>:</b> Auditing cabal projects <span class="badge bg-primary">beginner-friendly</span></li>
    <li><b><a href="https://github.com/haskell-hvr/regex-tdfa">regex-tdfa</a>:</b> Adding proper character class algebra</li>
    <li><b><a href="https://keid.haskell-game.dev/">Keid</a>:</b> Desktop rendering engine with game batteries included</li>
    <li><b><a href="https://ghc-proposals.readthedocs.io/en/latest/proposals/0370-modifiers.html">GHC Syntax for Modifiers</a>:</b> Working on the implementation of the syntax extension</li>
    <li><b><a href="https://github.com/lortabac/parameters">GHC Implicit Parameters</a>:</b> Working on the proposal, improving the design</li>
    <li><b><a href="https://errors.haskell.org/">GHC errors documentation</a>:</b> Improvements to documentation <span class="badge bg-primary">beginner-friendly</span></li>
    <li><b><a href="https://www.lean-lang.org/">Lean</a> Tutorial</b> at Fri 13:30</li>
    <li><b>ACMS:</b> Automatically Schema-Versioned Content Management System <span class="badge bg-primary">beginner-friendly</span></li>
    <li><b><a href="https://github.com/hellwolf/thsh">THSH</a>:</b> A "noDSL" approach to mixing shell scripting with Haskell programs using Template Haskell</li>
  </ul>
</div>
</div>

<a name="contact"></a>
<div class="container p-4">
  <h2>Contact</h2>
  <p>
    Want to keep in touch and receive updates about MuniHac? We have an

    <a class="text-nowrap" href="https://twitter.com/MuniHac" target="_tab"><i class="bi bi-twitter-x"></i> account</a>, a
    <a class="text-nowrap" href="https://groups.io/g/munihac" target="_tab"><i class="bi bi-envelope-fill"></i> mailing list</a>, and a
    <a class="text-nowrap" href="https://munihac.slack.com" target="_tab"><i class="bi bi-slack"></i> Slack workspace</a>.
  </p>

  <div class="d-flex flex-row flex-wrap justify-content-center mb-2">
  <!-- TODO: verify links -->
  <div class="mx-2 mb-2">
    <a class="btn btn-outline-primary rounded-pill px-3" href="https://twitter.com/MuniHac" target="_tab">
      <i class="bi bi-twitter-x"></i>
      Follow @MuniHac
    </a>
  </div>
  <div class="mx-2 mb-2">
    <a class="btn btn-outline-primary rounded-pill px-3" href="https://groups.io/g/munihac" target="_tab">
      <i class="bi bi-envelope-fill"></i>
      Subscribe to munihac@groups.io</a>
  </div>
  <div class="mx-2 mb-2">
    <a class="btn btn-outline-primary rounded-pill px-3" href="https://join.slack.com/t/munihac/shared_invite/zt-2i1v0wxev-M_DkjOxBpuX2B5CfPk8Nlg" target="_tab">
      <i class="bi bi-slack"></i>
      Join munihac.slack.com</a>
  </div>
</div>


  <p>You can reach us via email: <a href="mailto:munihac@tngtech.com">munihac@tngtech.com</a>.</p>
</div>

 <div class="bg-dark text-light">
  <div class="container p-4">
    <p class="text-center">
      Each participant will retain ownership of any and all intellectual and industrial property rights to his or her
      work created or used during the Hackathon.
    </p>
    <div class="text-center legal-notice">
      <a class="link-primary-light" href="impressum.html">Legal Notice / Impressum</a>
      <a href="https://www.tngtech.com/en/privacy.html" target="_blank" class="link-primary-light">Privacy Notice</a>
    </div>

    <div id="footer" class="footer">
      Site proudly generated by <a class="link-primary-light" href="http://github.com/jaspervdj/hakyll">hakyll</a>.
    </div>
  </div>
</div>

    <script src="./assets/bootstrap.bundle.min.js"></script>
    <script src="./js/scripts.js"></script>
    <script src="./js/load-iframe.js"></script>
  </body>
</html>
