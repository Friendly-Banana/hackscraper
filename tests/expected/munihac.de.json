[
    {
        "url": "https://munihac.de",
        "image": "",
        "name": "",
        "description": "",
        "date": "",
        "location": ""
    },
    {
        "url": "https://munihac.de",
        "image": "",
        "name": "HÃ©cate:\nEffect Systems in Practice",
        "description": "Exploring Haskell Language Server via the Cabal Plugin",
        "date": "October 11â13, 2024",
        "location": "Munich"
    }
]